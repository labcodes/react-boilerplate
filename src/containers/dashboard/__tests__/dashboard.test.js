import React from 'react';
import renderer from 'react-test-renderer';

import Dashboard from 'containers/dashboard';

describe('Dashboard [Snapshot]', () => {
    it('Renders correctly', () => {
        const tree = renderer.create(<Dashboard />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});

