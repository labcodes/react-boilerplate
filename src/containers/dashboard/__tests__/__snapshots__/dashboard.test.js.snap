// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard [Snapshot] Renders correctly 1`] = `
<h1
  className="sc-bdVaJa irjdWV"
>
  Dashboard works!
</h1>
`;
